x/avs/types/utils.go

```diff
@@ -0,0 +1,107 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// CommitPhase represents the phases of the Two-Phase Commit protocol
+type CommitPhase uint8
```

Comment:
Do we need both `type CommitPhase uint8` and the one defined in `tx.proto` as `enum Phase` ?